docs/developer/running-saleor/configuration.mdx

```diff
@@ -185,6 +185,19 @@ Controls [Django’s secret key](https://docs.djangoproject.com/en/3.0/ref/setti
 
 Controls the production assets mount path. Defaults to `/static/`.
 
+### `JWT_EXPIRATION_DELTA`
+
+Controls the time which defines how long the JWT authentication token will be valid. The value should be provided in seconds. Defaults to 5 minutes.
+
+
+### `JWT_REFRESH_EXPIRATION_DELTA`
+
+Controls the time which defines how long the JWT refresh token will be valid. The value should be provided in seconds. Default to 30 days.
+
+### `JWT_SECRET`
+
+Controls the key used to encode/decode JWT token.
```

Comment:
We should also say that it defaults to `SECRET_KEY` with a link to this value's description: https://docs.saleor.io/docs/developer/running-saleor/configuration#secret_key

Reply:
Yes! I forgot about this. Thank you 👮  !